context/app/static/js/components/collections/CollectionsSearchContext.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { Dispatch, PropsWithChildren, useMemo, useState } from 'react';
+import { createContext, useContext } from 'js/helpers/context';
+
+interface CollectionsSearchStateContext {
```

Comment:
If we foresee adding more data to this state, then this interface is fine. Otherwise, we could simplify it:

```suggestion
type CollectionsSearchStateContext = string;
```

Then the extra `useMemo` wrapper around the search state isn't necessary.

The same applies to actions, but I could see us including a `reset` action or something similar.

Reply:
Great point! I don't foresee adding any more actions/state variables to this in the near future, so I'll simplify both of these.